library/iojs

```diff
@@ -2,15 +2,24 @@
 
 1.0.4: git://github.com/iojs/docker-iojs@39ab80a90ff9addd935d3acf1f5bc3eff49d6ffc 1.0
```

Comment:
@tianon no, according to https://github.com/iojs/docker-iojs/pull/18#issuecomment-72789460 we will remove 1.0.x. Do you want to delay this PR for that?